nni/runtime/log.py

```diff
@@ -45,41 +180,7 @@ def init_logger() -> None:
         _init_logger_trial()
         return
 
-    _init_logger_standalone()
-
-    logging.getLogger('filelock').setLevel(logging.WARNING)
-
-_cli_log_initialized = False
-
-def init_logger_for_command_line() -> None:
-    """
-    Initialize logger for command line usage.
-    This means that NNI is used as "main function" rather than underlying library or background service,
-    so it should print log to stdout.
-
-    It is used by nnictl and `nni.experiment.Experiment`.
-
-    This function will get invoked after `init_logger()`.
-    """
-    global _cli_log_initialized
-    if not _cli_log_initialized:
-        _cli_log_initialized = True
-        colorful_formatter = Formatter(log_format, time_format)
-        colorful_formatter.format = _colorful_format
-        if '_default_' not in handlers:  # this happens when building sphinx gallery
-            _register_handler(StreamHandler(sys.stdout), logging.INFO)
-        handlers['_default_'].setFormatter(colorful_formatter)
-
-def start_experiment_log(experiment_id: str, log_directory: Path, debug: bool) -> None:
-    log_path = _prepare_log_dir(log_directory) / 'dispatcher.log'
-    log_level = logging.DEBUG if debug else logging.INFO
-    _register_handler(FileHandler(log_path), log_level, experiment_id)
-
-def stop_experiment_log(experiment_id: str) -> None:
-    if experiment_id in handlers:
-        handler = handlers.pop(experiment_id, None)
-        if handler is not None:
-            logging.getLogger().removeHandler(handler)
+    start_stdout_logging()
```

Comment:
What's the condition in which this will be triggered?

Reply:
On importing `nni`. We assume this has no effect "outside" NNI.